tools/wptrunner/wptrunner/browsers/webkit.py

```diff
@@ -43,6 +43,16 @@ def capabilities_for_port(server_config, **kwargs):
                  "certificateFile": kwargs["host_cert_path"]}
             ]
         }
+
+        # Manually map Selenium's key names to the corresponding W3C versions.
```

Comment:
Couldn't you just statically define a set of capabilities rather than still depending on the selenium defaults? Looks like the [DesiredCapabilites.WEBKITGTK capabilities](https://github.com/SeleniumHQ/selenium/blob/07a18746ff756e90fd79ef253a328bd7dfa9e6dc/py/selenium/webdriver/common/desired_capabilities.py#L124) are rather simple anyway:

Reply:
Yeah I'll do that.